packages/driver/types/internal-types.d.ts

```diff
@@ -52,6 +52,10 @@ declare namespace Cypress {
     projectRoot?: string
   }
 
+  interface RuntimeServerConfigOptions {
+    protocolEnabled: boolean
```

Comment:
I'd expect this to be added to the packages/config options as an internal cypress-config option

Reply:
I don't think this option belongs in packages/config as I view that package for options that you will find in the cypress config file and this is a runtime server config option. However, it does look like the other internal server options were added [here](https://github.com/cypress-io/cypress/blob/a40067bb3fe96912c2be9feed2557df247dadb4f/cli/types/cypress.d.ts#L3226), so I can move this one over there.

Reply:
packages/config contains run-time options for Cypress in addition to the cypress.configuration options. Its a single place to know what we are configuring and setting: https://github.com/cypress-io/cypress/blob/a40067bb3fe96912c2be9feed2557df247dadb4f/packages/config/src/options.ts#L464

Reply:
added option to the config: [3ac05a5](https://github.com/cypress-io/cypress/pull/26757/commits/3ac05a50cda60388d297a148c2c13531346c58b4)